deepchecks/nlp/utils/text_embeddings.py

```diff
@@ -0,0 +1,86 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2023 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Utils module for calculating embeddings for text."""
```

Comment:
There is a file called llm_utils, I think you can merge between them

Reply:
I took the function out of there. I think this is more organized this way (we discussed this 2 months ago actually)